docs/references/http-websocket-apis/public-api-methods/path-and-order-book-methods/book_changes.md

```diff
@@ -0,0 +1,125 @@
+---
+seo:
+    description: Get information on 
+labels:
+  - Decentralized Exchange
+  - Cross-Currency
+---
+# book_changes
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/ripple/rpc/BookChanges.h "Source")
+
+The {% code-page-name /%} method reports information about changes to the order books in the [decentralized exchange (DEX)](../../../../concepts/tokens/decentralized-exchange/index.md) compared with the previous ledger version. This may be useful for building "candlestick" charts.
+
+### Request Format
+
+An example of the request format:
+
+{% tabs %}
+
+{% tab label="WebSocket" %}
+```json
+{
+    "id": "example_book_changes",
+    "command": "book_changes",
+    "ledger_index": 88530953
+}
+```
+{% /tab %}
+
+{% tab label="JSON-RPC" %}
+```json
+{
+    "method": "{% $frontmatter.seo.title %}",
+    "params": [{
+      "ledger_index": 88530953
+    }]
+}
+```
+{% /tab %}
+
+{% tab label="Commandline" %}
+```sh
+#Syntax: book_changes [<ledger hash|id>]
+rippled book_changes 88530953
+```
+{% /tab %}
+
+{% /tabs %}
+
+[Try it! >](/resources/dev-tools/websocket-api-tool#book_changes)
+
+The request includes the following parameters:
+
+| Field          | Type             | Required? | Description |
+|:---------------|:-----------------|:----------|-------------|
+| `ledger_hash`  | [Hash][]         | No        | A 32-byte hex string for the ledger version to use. (See [Specifying Ledgers][]) |
+| `ledger_index` | [Ledger Index][] | No        | The [ledger index][] of the ledger to use, or a shortcut string to choose a ledger automatically. (See [Specifying Ledgers][]) |
+
+{% admonition type="warning" name="Known Issues" %}
+There are several known issue with this method in `rippled`.
+
+- You must specify the `ledger_index` or `ledger_hash` explicitly instead of using a default or a shortcut string. ([Issue #5034](https://github.com/XRPLF/rippled/issues/5034))
+- The response may be missing the `validated` field even when querying validated ledgers. ([#5035](https://github.com/XRPLF/rippled/issues/5035))
```

Comment:
The linked issue says this only is a problem using JSON-RPC. Should you mention that so people use alternative methods in the meantime?

Reply:
The recommended alternative in all cases is to query a Clio server, but I can modify it to be more specific here.